### PR TITLE
add: hindi translation of contributor guidelines

### PR DESCRIPTION
**Pull request by jaiakash**
Resolves - add hindi translation for contributor guidelines

 Changes: Added Hindi translation of contributor guidelines

#### PR Checklist

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [x] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

### CODE REVIEW

**Comment by jaiakash** [comment]
@TanviKumar @Divyansh013 can you please review this PR and let me know the changes if any.

Thanks !

File: contributor_docs/hi/contributor_guidelines.md [hunk 1]
```diff
@@ -0,0 +1,528 @@
+# योगदानकर्ता दिशानिर्देश
+
+योगदानकर्ता दिशानिर्देशों में आपका स्वागत है! यह दस्तावेज़ उन नए योगदानकर्ताओं के लिए है जो p5.js में कोड का योगदान करना चाहते हैं, ऐसे योगदानकर्ता जो कुछ तकनीकी चरणों पर अपनी यादें ताज़ा करना चाहते हैं, या p5.js में कोड योगदान के साथ कुछ और करना चाहते हैं।
+
+यदि आप p5.js रिपॉजिटरी के बाहर योगदान करना चाहते हैं (ट्यूटोरियल लिखना, कक्षाओं की योजना बनाना, कार्यक्रमों का आयोजन करना), तो कृपया इसके बजाय अन्य प्रासंगिक पृष्ठों पर एक नज़र डालें। प्रबंधकों या अनुरक्षकों को समस्याएँ की समीक्षा करने और अनुरोधों को खींचने के संबंध में [कार्यपालिका दिशानिर्देश](https://github.com/processing/p5.js/blob/main/contributor_docs/steward_guidelines.md) अधिक उपयोगी लग सकते हैं।
```
Comment (Divyansh013) [inline contributor_docs/hi/contributor_guidelines.md:5]: hello, this whole text looks too typical can we have some alternatives here?

File: contributor_docs/hi/contributor_guidelines.md [hunk 2]
```diff
@@ -0,0 +1,528 @@
...
+  - [पुल रिक्वेस्ट बनाना](#पुल-रिक्वेस्ट-बनाना)
+    - [पुल रिक्वेस्ट के बारे में जानकारी](#पुल-रिक्वेस्ट-के-बारे-में-जानकारी)
+    - [शीर्षक](#शीर्षक)
+    - [हल](#हल)
+    - [परिवर्तन](#परिवर्तन)
+    - [परिवर्तन के स्क्रीनशॉट](#परिवर्तन-के-स्क्रीनशॉट)
+    - [पीआर-चेकलिस्ट](#पीआर-चेकलिस्ट)
+    - [रिबेस और रेसोल्वे कनफ्लिक्ट](#रिबेस-और-रेसोल्वे-कनफ्लिक्ट)
+  - [चर्चा करें और संशोधन करें](#चर्चा-करें-और-संशोधन-करें)
+
```
Comment (Divyansh013) [inline contributor_docs/hi/contributor_guidelines.md:46]: Hi, have you checked all the links of table of contents if they are working?

File: contributor_docs/hi/contributor_guidelines.md [hunk 3]
```diff
@@ -0,0 +1,528 @@
...
+# समस्याएँ के बारे में सब कुछ
+
+p5.js के गिटहब रिपॉजिटरी (संक्षेप में रेपो) पर अधिकांश गतिविधि समस्याएँ में होती है, जो आपकी योगदान यात्रा शुरू करने के लिए एक शानदार जगह है।
+
+## समस्याएँ क्या हैं?
+
+![p5.js लाइब्रेरी गिटहब रिपॉजिटरी का एक क्रॉप्ड स्क्रीनशॉट, केवल ऊपरी दाएं कोने की सामग्री दिखा रहा है। स्क्रीनशॉट के शीर्ष पर मुद्दे टैब के आसपास एक लाल बॉक्स बनाया गया है।](../images/issues-tab.png)
+
+"समस्या" गिटहब पर एक पोस्ट के लिए एक सामान्य नाम है जो किसी समस्या को वर्णित करने का उद्देश्य रखता है। यह समस्या एक त्रुटि रिपोर्ट, नई फीचर जोड़ने का अनुरोध, एक चर्चा, या p5.js पुस्तकालय विकास से संबंधित किसी भी पोस्ट के रूप में काम कर सकती है। प्रत्येक समस्या के नीचे किसी भी गिटहब खाते वाले व्यक्ति, सहित बॉट्स, द्वारा टिप्पणियाँ जोड़ी जा सकती हैं! यह वह स्थान है जहां योगदानकर्ताओं ने परियोजना के विकास से संबंधित विषयों पर चर्चा करते हैं।
```
Comment (Divyansh013) [inline contributor_docs/hi/contributor_guidelines.md:57]: use "लाइब्रेरी" for library

File: contributor_docs/hi/contributor_guidelines.md (hunk 2 above)
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:46]: Yeah these links are working fine.

File: contributor_docs/hi/contributor_guidelines.md [hunk 4]
```diff
@@ -0,0 +1,528 @@
...
+
+जबकि एक समस्या कई विभिन्न कारणों के लिए खोली जा सकती है, हम आमतौर पर केवल p5.js स्रोत कोड के विकास पर चर्चा करने के लिए समस्याएँ का उपयोग करते हैं। अपने कोड की डीत्रुटििंग, अपने परियोजना में सहयोगी निमंत्रण करने, या अन्य असंबंधित विषयों जैसे विषयों पर हम केवल [फोरम](https://discourse.processing.com) या अन्य प्लेटफ़ॉर्मों जैसे [डिस्कोर्ड](https://discord.gg/SHQ8dH25r9) पर चर्चा करते हैं।
```
[the remark this answers is not in the document]
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:59]: डीत्रुटििंग is debugging. Debugging has no exact hindi word, so i have used पुन: निरीक्षण for debugging

File: contributor_docs/hi/contributor_guidelines.md (hunk 3 above)
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:57]: Done

File: contributor_docs/hi/contributor_guidelines.md [hunk 5]
```diff
@@ -0,0 +1,528 @@
...
+
+हमने आपको गिटहब समस्या होनी चाहिए या कहीं और पोस्ट करनी चाहिए को निर्धारित करने में मदद करने के लिए आसान-से-उपयोग निर्देशिका बनाई हैं!
+
+## समस्या खाका
+
+p5.js के समस्या खाका उपाध्यक्षों और रखरखावकर्ताओं को समस्याओं को समझने और समीक्षा करने में सुविधा प्रदान करते हैं। ये आपको भी उचित समस्या को दर्ज करने और त्वरित उत्तर प्राप्त करने में सहायक होते हैं।
+
+![गिटहब पर एक मामले की जैसे दिखने वाली एक उदाहरण स्क्रीनशॉट। स्क्रीनशॉट में मुद्दे का शीर्षक "सफारी में चेतावनी लॉग हो रही है जब 2डी मोड में फ़िल्टर शेडर का उपयोग किया जाता है #6597" है।](../images/github-issue.png)
+
+नई समस्या दर्ज करने के लिए, पहले "Issues" टैब पर जाएं पी5.js रेपो पर और फिर दाएं ओर "New issue" बटन पर क्लिक करें। आपको कई विभिन्न विकल्पों के साथ प्रस्तुत किया जाएगा, जिसमें से प्रत्येक किसी संबंधित समस्या खाका का अनुसरण करता है या आपको आपके प्रश्न को दर्ज करने के लिए उचित स्थान पर पुनःनिर्देशित करता है। हम सुझाव देते हैं कि आप प्रस्तुत किए गए सभी विकल्पों में से सबसे संबंधित विकल्प का चयन करें ताकि आपकी समस्या को सही ध्यान और समय से संबोधित किया जा सके।
```
[the remark this answers is not in the document]
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:69]: Done

File: contributor_docs/hi/contributor_guidelines.md [hunk 6]
```diff
@@ -0,0 +1,528 @@
...
+
+</td>
+
+</tr>
+
+<tr>
+
+<td>
+
+पता बार में, `chrome://version` पर नेविगेट करें
```
[the remark this answers is not in the document]
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:111]: Changed the sentence to sound natural in hindi

File: contributor_docs/hi/contributor_guidelines.md [hunk 7]
```diff
@@ -0,0 +1,528 @@
...
+
+शीर्ष पट्टी के नीचे “Safari” मेनू में से, “About Safari” का चयन करें
+
+</td>
+
+</tr>
+
+</table>
+
+4. *ऑपरेटिंग सिस्टम* - यदि संभव हो तो आपको ओएस संस्करण संख्या शामिल करनी चाहिए, उदाहरण के लिए, `macOS 12.5`। कुछ त्रुटि OS व्यवहार से उत्पन्न हो सकते हैं।
```
[the remark this answers is not in the document]
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:131]: for OS, i have written OS as ओएस

[the remark this answers is not in the document]
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:131]: for OS, i have written OS as ओएस

File: contributor_docs/hi/contributor_guidelines.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:111]: Changed the sentence to sound natural in hindi

File: contributor_docs/hi/contributor_guidelines.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md]: done

**Comment by jaiakash** [comment]
Hi @SanjaySinghRajpoot, i have updated the PR with the suggested changes. Some changes are left, Can you update the PR for same.

File: contributor_docs/hi/contributor_guidelines.md [hunk 8]
```diff
@@ -0,0 +1,528 @@
...
+5. *इसे पुन: प्रस्तुत करने के चरण* - यह संभवतः साझा करने के लिए सबसे महत्वपूर्ण जानकारी है। आप जो त्रुटि देख रहे हैं उसकी प्रतिकृति बनाने के लिए आपको विस्तृत चरणों की सूची बनानी चाहिए। एक बुनियादी नमूना कोड साझा करना जो समस्या को प्रदर्शित करता है, उस व्यक्ति के लिए बहुत मददगार साबित हो सकता है जो उस त्रुटि को दोहराना चाहता है जिसका आप सामना कर रहे हैं और समाधान तैयार करना शुरू कर सकते हैं।
```
[the remark this answers is not in the document]
Comment (SanjaySinghRajpoot) [reply contributor_docs/hi/contributor_guidelines.md:132]: प्रतिदर्श is used now in place of नमूना

File: contributor_docs/hi/contributor_guidelines.md [hunk 9]
```diff
@@ -0,0 +1,528 @@
...
+
+### ["चर्चा"](https://github.com/processing/p5.js/issues/new?assignees=\&labels=Discussion\&projects=\&template=discussion.yml)
+
+इस खाका का उपयोग तब किया जाता है जब आप जो मुद्दा दाखिल कर रहे हैं वह किसी भी तरह से उपरोक्त में से किसी में फिट नहीं बैठता है। उपरोक्त किसी भी खाका में फिट न होने वाला मुद्दा व्यवहार में अपेक्षाकृत दुर्लभ होना चाहिए। उदाहरण के लिए, p5.js में एक विशिष्ट वेब एपीआई सुविधा को अपनाने के बारे में चर्चा को [नई सुविधा अनुरोध](https://github.com/processing/p5.js/blob/main/contributor_docs/contributor_guidelines.md#new-feature-request); विभिन्न रंग कार्यों में एक अतिरिक्त रंग मोड जोड़ने के बारे में चर्चा को [सुविधा संवर्धन](https://github.com/processing/p5.js/blob/main/contributor_docs/contributor_guidelines.md#existing-feature-enchancement); आपके द्वारा आयोजित किए जा रहे स्थानीय रचनात्मक कोडिंग कार्यक्रम के बारे में एक घोषणा मंच पर पोस्ट की जानी चाहिए और यदि आप समर्थन या प्रचार की तलाश में हैं तो प्रोसेसिंग फाउंडेशन से संपर्क करें; वगैरह।
```
[the remark this answers is not in the document]
Comment (SanjaySinghRajpoot) [reply contributor_docs/hi/contributor_guidelines.md:173]: उपयुक्त is used in place of suit

File: contributor_docs/hi/contributor_guidelines.md [hunk 10]
```diff
@@ -0,0 +1,528 @@
...
+
+किसी चर्चा मुद्दे को खोलते समय, आप अतिरिक्त प्रासंगिक लेबल जोड़ने के लिए साइड पैनल पर "लेबल" पैनल का उपयोग कर सकते हैं ताकि आप अपने मुद्दे को संबंधित क्षेत्र में साइनपोस्ट कर सकें। टेम्प्लेट स्वयं न्यूनतम पाठ फ़ील्ड है। उदाहरण चर्चा मुद्दे के लिए आप [यह लिंक](https://github.com/processing/p5.js/issues/6517) देख सकते हैं।
```
[the remark this answers is not in the document]
Comment (SanjaySinghRajpoot) [reply contributor_docs/hi/contributor_guidelines.md:175]: it's alternative will end up being more complex

File: contributor_docs/hi/contributor_guidelines.md [hunk 11]
```diff
@@ -0,0 +1,528 @@
...
+  npm test
+  ```
+
+8. यदि आप नई सुविधाएँ जोड़ने या सुविधा बढ़ाने पर काम कर रहे हैं तो कोई भी इकाई परीक्षण जोड़ें।
+
+9. एक बार हो जाने के बाद, आप परिवर्तन कर सकते हैं और एक [पुल अनुरोध](https://p5js.org/contributor-docs/#/./contributor_guidelines?id=pull-requests) बना सकते हैं।
+
+## गिटहब संपादन कार्यक्षमता का उपयोग करना
+
+गिटहब वेब इंटरफ़ेस पर किसी फ़ाइल को देखते समय, आप जिस फ़ाइल को देख रहे हैं उसकी सामग्री के शीर्ष के पास एक पेंसिल आइकन बटन होगा। यह बटन गिटहब द्वारा प्रदान की गई एक सुविधाजनक संपादन सुविधा है जो उन कई प्रक्रियाओं को सरल बनाती है जिन्हें हम नीचे कवर करेंगे और इसका उपयोग आपके द्वारा देखी जा रही फ़ाइल में त्वरित और सरल संपादन करने के लिए किया जा सकता है।
```
[the remark this answers is not in the document]
Comment (SanjaySinghRajpoot) [reply contributor_docs/hi/contributor_guidelines.md:241]: done

File: contributor_docs/hi/contributor_guidelines.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (TanviKumar) [reply contributor_docs/hi/contributor_guidelines.md]: So some parts have "OS" as is, so use it consistently

File: contributor_docs/hi/contributor_guidelines.md [hunk 12]
```diff
@@ -0,0 +1,530 @@
...
+git add .
+```
+
+3. गिट में परिवर्तन करने के लिए, निम्न कमांड चलाएँ।
+
+```
+git commit -m "[your_commit_message]"
+```
+
+`[your_commit_message]` को एक प्रासंगिक प्रतिबद्ध संदेश के साथ प्रतिस्थापित किया जाना चाहिए जो सामान्य बयानों से बचते हुए परिवर्तनों का वर्णनात्मक हो। उदाहरण के लिए, 'डॉक्यूमेंटेशन फिक्स 1' कहने के बजाय, 'सर्कल() फ़ंक्शन में दस्तावेज़ उदाहरण जोड़ें' कहें।
```
Comment (Divyansh013) [inline contributor_docs/hi/contributor_guidelines.md:367]: can we make this text a bit more simplier?

File: contributor_docs/hi/contributor_guidelines.md [hunk 13]
```diff
@@ -0,0 +1,530 @@
...
+
+## कोड मानक
+
+p5.js का कोड मानक या कोड शैली [ESLlint](https://eslint.org/) द्वारा लागू की जाती है। किसी भी गिट कमिट और पुल अनुरोध को स्वीकार करने से पहले उसे लिंटिंग से गुजरना होगा। आपके लिए सही कोडिंग मानक का पालन करने का सबसे आसान तरीका आपके टेक्स्ट एडिटर के लिए लिंटिंग एरर हाइलाइटिंग (अधिकांश लोकप्रिय टेक्स्ट संपादकों के लिए उपलब्ध) के साथ उपलब्ध ESLint प्लगइन का उपयोग करना है।
+
+## सॉफ़्टवेयर डिज़ाइन सिद्धांत
+
+p5.js की किसी भी विशेषता पर काम करते समय, p5.js के डिज़ाइन सिद्धांतों को ध्यान में रखना महत्वपूर्ण है। हमारी प्राथमिकताएँ अन्य परियोजनाओं की प्राथमिकताओं से भिन्न हो सकती हैं, इसलिए यदि आप किसी भिन्न परियोजना से आ रहे हैं, तो हम अनुशंसा करते हैं कि आप स्वयं को p5.js के डिज़ाइन सिद्धांतों से परिचित करा लें।
+
+- **पहुंच** हम सबसे पहले पहुंच को प्राथमिकता देते हैं, और हमारे द्वारा लिए गए निर्णयों में इस बात को ध्यान में रखना चाहिए कि यह ऐतिहासिक रूप से हाशिए पर रहने वाले समूहों तक पहुंच कैसे बढ़ाता है। हमारे पहुंच विवरण में इसके बारे में और पढ़ें।
```
Comment (Divyansh013) [inline contributor_docs/hi/contributor_guidelines.md:401]: "उपलब्धता" will be better for accessibility

File: contributor_docs/hi/contributor_guidelines.md [hunk 14]
```diff
@@ -0,0 +1,530 @@
...
+- **शुरुआती अनुकूल** p5.js एपीआई का लक्ष्य शुरुआती कोडर्स के लिए अनुकूल होना है, जो अत्याधुनिक HTML5/Canvas/DOM एपीआई के साथ परस्पर संवादात्मक और दृश्य वस्तु वेब सामग्री बनाने में कम बाधा प्रदान करता है।
+- **शैक्षिक** p5.js एक एपीआई और पाठ्यक्रम पर केंद्रित है जो शैक्षिक उपयोग का समर्थन करता है, जिसमें सहायक उदाहरणों के साथ एपीआई का पूरा संदर्भ, साथ ही शिक्षक और प्रतिदर्श कक्षा पाठ्यक्रम शामिल हैं जो मुख्य रचनात्मक कोडिंग सिद्धांतों को स्पष्ट रूप से पेश करते हैं।
+- **जावास्क्रिप्ट और उसका समुदाय** p5.js का उद्देश्य उचित जावास्क्रिप्ट डिज़ाइन पैटर्न और उपयोग को मॉडलिंग करके शुरुआती लोगों के लिए वेब विकास प्रथाओं को अधिक सुलभ बनाना है, जबकि आवश्यक होने पर उनका सार तैयार करना है। एक ओपन-सोर्स लाइब्रेरी के रूप में, p5.js में इसके निर्माण, दस्तावेज़ीकरण और प्रसार में व्यापक जावास्क्रिप्ट समुदाय भी शामिल है।
+- **प्रसंस्करण और उसका समुदाय** p5.js प्रसंस्करण भाषा और उसके समुदाय से प्रेरित है और इसका उद्देश्य प्रसंस्करण जावा से जावास्क्रिप्ट में परिवर्तन को आसान और स्पष्ट बनाना है।
```
Comment (Divyansh013) [inline contributor_docs/hi/contributor_guidelines.md:405]: keep processing as processing only. don't translate

File: contributor_docs/hi/contributor_guidelines.md [hunk 15]
```diff
@@ -0,0 +1,530 @@
...
+
+
+### पीआर चेकलिस्ट
+
+इसमें कुछ प्रासंगिक चेकलिस्ट आइटम शामिल हैं जिन्हें आपको अपने परिवर्तनों के लिए प्रासंगिक होने पर `[ ]` को `[x]` से प्रतिस्थापित करके टिक करना चाहिए।
+
+एक बार हो जाने पर, "Create pull request." पर क्लिक करें।
+
+
+### रिबेस और रेसोल्वे कनफ्लिक्ट
```
Comment (Divyansh013) [inline contributor_docs/hi/contributor_guidelines.md:478]: check resolve spelling

**Review by Divyansh013 — CHANGES_REQUESTED** [review]

File: contributor_docs/hi/contributor_guidelines.md (hunk 14 above)
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:405]: done

File: contributor_docs/hi/contributor_guidelines.md (hunk 13 above)
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:401]: Done

Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:401]: done

File: contributor_docs/hi/contributor_guidelines.md (hunk 15 above)
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:478]: i have used विवाद for conflict

File: contributor_docs/hi/contributor_guidelines.md (hunk 12 above)
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:367]: Done

File: contributor_docs/hi/contributor_guidelines.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md]: Now, there is no occurrence of OS, changed to ओएस

File: contributor_docs/hi/contributor_guidelines.md [hunk 16]
```diff
@@ -0,0 +1,528 @@
...
+
+![हरे "नए अंक" बटन के साथ गिटहब रिपॉजिटरी के अंक पृष्ठ का क्रॉप किया गया स्क्रीनशॉट, जिसके चारों ओर एक लाल बॉक्स हाइलाइट किया गया है।](../images/new-issue.png)
+
+### ["कोई बग मिला"](https://github.com/processing/p5.js/issues/new?assignees=\&labels=Bug\&projects=\&template=found-a-bug.yml)
+
+जब आप p5.js में संभावित गलत व्यवहार का सामना करते हैं या दस्तावेज़ में वर्णित अनुसार व्यवहार नहीं करते हैं, तो [इस खाका](https://github.com/processing/p5.js/issues/new?assignees=\&labels=Bug\&projects=\&template=found-a-bug.yml). कृपया ध्यान दें कि यदि आप अपने स्केच को डीत्रुटि करने का प्रयास कर रहे हैं और आपको लगता है कि यह आपके कोड में कोई समस्या हो सकती है, तो आपको पहले [डिस्कोर्स फोरम](https://discourse.processing.org) पर पूछना चाहिए।
```
[the remark this answers is not in the document]
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md]: done

File: contributor_docs/hi/contributor_guidelines.md (hunk 1 above)
Comment (jaiakash) [reply contributor_docs/hi/contributor_guidelines.md:5]: done

**Review by TanviKumar — APPROVED** [review]

**Review by Divyansh013 — APPROVED** [review]

**Review by SanjaySinghRajpoot — APPROVED** [review]

**Comment by jaiakash** [comment]
Hi @Qianqianye this PR is approved by stewards. Can you review and merge it if there are you no changes. Thanks in advance.

**Comment by Qianqianye** [comment]
Thanks @jaiakash for working on it! Thanks @Divyansh013 @TanviKumar @SanjaySinghRajpoot for reviewing!
@all-contributors please add @jaiakash for translation

**Pull request merged**